engine/src/dskmac.cpp

```diff
@@ -567,7 +567,13 @@ static void parseSerialControlStr(MCStringRef setting, struct termios *theTermio
         {
             integer_t baudrate;
             /* UNCHECKED */ MCStringToInteger(*t_value, baudrate);
-			if (baudrate == 57600)
```

Comment:
As mentioned in another comment - it appears you can just do baud=baudrate on macOS (not that arbitrary values will necessary work - but stty works that way).